loganharbour/civet
# Check for recipes before sleeping in scheduler

- Bump django from 2.2.10 to 2.2.13
- Adding list of modulecmds to try.
- Sort client listing by active (seen in < 60 s) first
- Try looking for modulecmd in path as a last resort
- Make PR work on meowbuild
- Fix python syntax mistake
- A small change... just for new recipes
- Trigger another build...
- Trigger a build
- testing fix
- cleaned text up for button fix
- Delete .DS_Store
- Delete .DS_Store
- Delete .DS_Store
- Testing potential fix to tests
- print for test
- Uncapitalized one letter
- modularized login labels for future use
- removed hardcoding
- Bump django from 2.2.13 to 2.2.18
- Bump django from 2.2.18 to 2.2.20
- Bump django-debug-toolbar from 2.0 to 2.2.1
- Bump urllib3 from 1.25.6 to 1.25.8
- Fix Issue creation for GitLab instances
- Bump urllib3 from 1.25.8 to 1.26.5
- Reverting urllib3 update
- Bump django from 2.2.20 to 2.2.21
- Bump django from 2.2.21 to 2.2.24
- Initial scheduler changes
- untested mods to np's work
- add logging
- use uwsgi worker ids instead of silly locking
- only import uwsgi if it exists - because sometimes it doesn't when we run scripts and stuff outside the production civet server context
- enable canceling jobs older than a certain number of minutes
- store last scheduled time in db so wsgi restarts don't make us 'forget' when last runs were
- by default don't run jobs immediately when they first get a cron schedule - wait for *next* run time
- update requirements.txt
- set scheduling interval at 5 minutes
- whitespace fixes?
- removed daemon-schedule, fixed imports
- removed superfluous app
- Clean up PR
- Clean up/fix timing in the scheduler
- Check for recipes before sleeping
